src/Data/UTxO/Transaction/Cardano/Helpers.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+
+{-# OPTIONS_HADDOCK prune #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-} -- needed for {#- HLINT ... #-}
+
+module Data.UTxO.Transaction.Cardano.Helpers
```

Comment:
Having these as a separate module makes sense, but these have nothing to do with UTxO nor cardano ;)

Reply:
this is probably temporary in the end as we will remove Byron stuff, and this could be merged with Shelley.